views/partners.handlebars

```diff
@@ -88,6 +88,12 @@
           <a href="https://cloud.google.com/blog/products/maps-platform/how-google-maps-platform-community-responding-covid-19-visualizing-helpful-info" target="_blank" class="partner-logo-container">
             <img src="/images/logos-Media/Google_Cloud.png" alt="Google Cloud Logo">
           </a>
+          <a href="https://www.businessinsider.com/how-you-can-help-donate-face-masks-doctors-nurses-coronavirus-2020-3" target="_blank" class="partner-logo-container">
+            <img src="/images/logos-Media/Business Insider.png" alt="Business Insider Logo">
+          </a>
+          <a href="https://news.westernu.ca/2020/04/students-join-global-effort-to-solve-ppe-shortage/" target="_blank" class="partner-logo-container">
+            <img src="/images/logos-Media/Western_University.gif" alt="Western University Logo">
```

Comment:
@linderr do we really want to use a gif here?

Reply:
Lol no. The official university communications logo was locked behind a log-in & this is the troll file they made publicly available. I’ll look for a better extension later